packages/ui/components/Avatars/index.tsx

```diff
@@ -12,21 +13,21 @@ export const PersonAvatar: React.FC<PersonAvatarProps> = ({ personId }) => {
 };
 
 interface ProfileAvatar {
-  userId: string;
   imageUrl?: string | null;
+  email: string;
 }
 
-export const ProfileAvatar: React.FC<ProfileAvatar> = ({ userId, imageUrl }) => {
-  if (imageUrl) {
-    return (
-      <Image
-        src={imageUrl}
-        width="40"
-        height="40"
-        className="h-10 w-10 rounded-full object-cover"
-        alt="Avatar placeholder"
-      />
-    );
-  }
-  return <Avatar size={40} name={userId} variant="bauhaus" colors={colors} />;
+export const ProfileAvatar: React.FC<ProfileAvatar> = ({ imageUrl, email }) => {
+  const hashedEmail = createHash("sha256").update(email).digest("hex");
```

Comment:
@wellyshen CodeRabbit is partially right. I think we should force the email to be lowercase first since that's what the docs say: https://docs.gravatar.com/api/avatars/hash/

(we likely already have the email addresses in our system already lowercase but makes sense to also have this here since there might be non-lowercase emails in the db)